types/xrm/package.json

```diff
@@ -2,8 +2,10 @@
     "private": true,
     "name": "@types/xrm",
     "version": "9.0.9999",
+    "nonNpm": true,
```

Comment:
```suggestion
    "nonNpm": "conflict"
```

Reply:
(and for the others)

Reply:
Updated (and [made a PR to update the docs](https://github.com/microsoft/DefinitelyTyped-tools/pull/1119))